tests/ReactTestApp/ios/ReactTestAppTests/ReactTestAppTests.m

```diff
@@ -267,16 +240,16 @@ - (void)invokeMethod:(NSString *)method {
 
 @end
 
-#if RCT_DEV
-@implementation RealmReactChromeTests
-
-+ (Class)executorClass {
-    return [RCTWebSocketExecutor class];
-}
-
-+ (NSString *)classNameSuffix {
-    return @"_Chrome";
-}
-
-@end
-#endif
+//#if RCT_DEV
```

Comment:
If you're removing the chrome tests, there's some more code in defaultTestSuites related to renaming the chrome version of the tests that should go away.

Reply:
Found it.  Thanks!